Code/GraphMol/RGroupDecomposition/RGroupUtils.h

```diff
@@ -93,6 +93,23 @@ RDKIT_RGROUPDECOMPOSITION_EXPORT std::string toJSON(
 RDKIT_RGROUPDECOMPOSITION_EXPORT std::string toJSON(
     const RGroupColumns &rgr, const std::string &prefix = "");
 
+//! Relabel dummy atoms bearing an R-group mapping (as
+/// atom map number, isotope or MDLRGroup label) such that
+/// they will be displayed by the rendering code as R# rather
+/// than #*, *:#, #*:#, etc. By default, only the MDLRGroup label
+/// is retained on output; this may be configured through the
+/// outputLabels parameter.
+/// In case there are multiple potential R-group mappings,
+/// the priority on input is Atom map number > Isotope > MDLRGroup.
+/// The inputLabels parameter allows to configure which mappings
+/// are taken into consideration.
+RDKIT_RGROUPDECOMPOSITION_EXPORT void relabelMappedDummies(
+    ROMol &mol,
+    RGroupLabelling inputLabels = static_cast<RGroupLabelling>(AtomMap |
```

Comment:
having to include the static_cast here is pretty ugly.
I think I'd prefer that both this parameter and `outputLabels` be unsigned ints, like the `labels` and `rgroupLabeling` members of `RGroupDecompositionParameters`.
What's your take on that?

Reply:
I agree.